optimum/utils/import_utils.py

```diff
@@ -64,12 +84,9 @@ def _is_package_available(pkg_name: str, return_version: bool = False) -> Union[
 _diffusers_available, _diffusers_version = _is_package_available("diffusers", return_version=True)
 _transformers_available, _transformers_version = _is_package_available("transformers", return_version=True)
 _torch_available, _torch_version = _is_package_available("torch", return_version=True)
-
-# importlib.metadata.version seem to not be robust with the ONNX Runtime extensions (`onnxruntime-gpu`, etc.)
-_onnxruntime_available = _is_package_available("onnxruntime", return_version=False)
-
-# TODO : Remove
-torch_version = version.parse(importlib.metadata.version("torch")) if _torch_available else None
+_onnxruntime_available, _onnxruntime_version = _is_package_available(
+    "onnxruntime", return_version=True, alt_pkg_names=["onnxruntime-gpu", "onnxruntime-training"]
+)
```

Comment:
might be interesting to do the same for tensorflow

Reply:
yes the same method should work for tf distributions.